app/assets/stylesheets/application.scss

```diff
@@ -749,6 +749,13 @@ span.inactive {
 
 .o-65 { opacity: 0.65; }
 
+.card-chip-container {
```

Comment:
You can move this into the '.card-chip' class and use a [parent-selector](https://sass-lang.com/documentation/style-rules/parent-selector/) to organize this

It will look something like this
```
.card-chip {
  ...
  &.container {
    ....
  }
}
```